ajax/libs/rpg-awesome

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rpg-awesome",
+  "filename": "rpg-awesome.min.css",
+  "description": "A fantasy themed font and CSS toolkit.",
+  "keywords": [
+    "rpg",
+    "font",
+    "awesome",
+    "css",
+    "toolkit"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nagoshiashumari/Rpg-Awesome.git"
+  },
+  "authors": [
+    "Daniela Howe",
+    "Ivan Montiel <idmontie@gmail.com>"
+  ],
+  "license": "MIT",
```

Comment:
There is two licenses we need to add in this file, as the README said: https://github.com/nagoshiashumari/Rpg-Awesome#license.
Let's modify as the followings:
```js
  "licenses": [
    "MIT",
    "OFL-1.1"
  ]
```
  Thanks.